Sources/PublishCLICore/WebsiteRunner.swift

```diff
@@ -55,6 +65,13 @@ private extension WebsiteRunner {
         catch { throw CLIError.outputFolderNotFound }
     }
 
+    /// The major version number of the current system's Python install (whatever Python responds to the `python` command)
```

Comment:
No documentation needed here either, it isn't a public-facing API, and it should be very self-explaining.